sonar-slang-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1125.html

```diff
@@ -0,0 +1 @@
+<p>Redundant Boolean literals should be removed from expressions to improve readability.</p>
```

Comment:
For this rule, I believe we should try to provide some code examples. In some cases, what this rule catches is not obvious.